lib/v2/screens/profile_screens/profile/profile_screen.dart

```diff
@@ -27,14 +30,40 @@ class ProfileScreen extends StatelessWidget {
               case PageState.failure:
                 return const FullPageErrorIndicator();
               case PageState.success:
-                return ListView(
-                  children: <Widget>[
-                    const ProfileHeader(),
-                    const DividerJungle(thickness: 2),
-                    const ProfileMiddle(),
-                    const DividerJungle(thickness: 2),
-                    const ProfileBottom(),
-                  ],
+                return Scaffold(
+                  appBar: AppBar(
+                    title: BlocBuilder<ProfileBloc, ProfileState>(builder: (context, state) {
+                      return DropdownButton<bool>(
+                        value: true,
+                        icon: const Icon(Icons.keyboard_arrow_down_sharp, color: AppColors.white),
+                        underline: const SizedBox.shrink(),
+                        hint: Text(state.profile?.account ?? '', style: Theme.of(context).textTheme.headline6),
+                        items: [],
+                        onTap: () {
+                          // For the onTap to work there must be at least one item.
+                        },
+                      );
+                    }),
+                    actions: [
+                      IconButton(
+                        icon: SvgPicture.asset(
+                          'assets/images/wallet/app_bar/scan_qr_code_icon.svg',
+                          height: 30,
+                          width: 2000,
```

Comment:
2000? why?

Reply:
To be honest I just copied from the existing onboarding design but now you mention it's weird.
I'll do a few tests to see if this value is really necessary.